docs/source/howto/plugins_install.rst

```diff
@@ -0,0 +1,65 @@
+.. _how-to:plugins-install:
+
+**********************
+How to install plugins
+**********************
+
+The functionality of AiiDA can be extended through plugins.
+There are various types of functionality that can be extended, such as new :ref:`data types<how-to:data:plugin>`, :ref:`calculation plugins<how-to:plugin-codes>` and much more.
+Multiple plugins can be bundled together and distributed in a :ref:`plugin package<how-to:plugins-develop>`.
+The `AiiDA plugin registry <https://aiidateam.github.io/aiida-registry>`_ gives an overview of public plugin packages.
+
+Installing an AiiDA plugin package is done with `pip <https://pypi.org/project/pip/>`_.
+If the package is distributed via the `Python Package Index (PyPI) <https://pypi.org/search/?q=aiida>`_ you can install it as follows:
+
+.. code-block:: console
+
+    $ pip install aiida-plugin-name
+
+A package can also be installed from the source code.
+In this case, instead of passing the plugin package name, pass the name of the directory that contains the ``setup.py`` file.
+For example, if the code is available through a Git repository:
+
+.. code-block:: console
+
+    $ git clone https://github.com/aiidateam/aiida-diff
+    $ pip install aiida-diff
+
+Each time when you install a new plugin package you should make sure to run the following command to let AiiDA know about the new plugins that come with it:
+
+.. code-block:: console
+
+    $ reentry scan
+
+If you forget to run this command, AiiDA will not be able to find the plugins.
+
+.. note::
+    The reentry cache can also be updated from python when access to the commandline is not available (e.g. in Jupyter notebooks).
+
+    .. code-block:: python
+
+        from reentry import manager
+        manager.scan(group_re='aiida')
+
```

Comment:
This is not really critical (actually you could just run `!reentry scan`). Anyway, the important thing to emphasise is the one above - so I would put the note above (on the fact `reentry scan` is important)